Quadriga/src/main/java/edu/asu/spring/quadriga/service/transformation/ITransformationSaveService.java

```diff
@@ -0,0 +1,25 @@
+package edu.asu.spring.quadriga.service.transformation;
+
+import java.io.IOException;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+
+import edu.asu.spring.quadriga.domain.workspace.ITextFile;
+import edu.asu.spring.quadriga.domain.workspace.ITransformationFile;
+import edu.asu.spring.quadriga.exceptions.FileStorageException;
+import edu.asu.spring.quadriga.utilities.IFileSaveUtility;
+
+public interface ITransformationSaveService{
+
+    /**
+     * @param txtFile
+     *            Text File Object to be saved
```

Comment:
this is not very informative. what does the method exactly do? there is no file object. what do you mean?